pkg/lib/serverTime.js

```diff
@@ -321,13 +302,109 @@
                     console.warn("failed to load time servers", error);
                     return result;
                 });
-    };
+    }
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17992-20221206-194101-a7fca0c4-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)